doozer/doozerlib/distgit.py

```diff
@@ -1286,6 +1289,48 @@ def update_build_db(self, success_flag, task_id=None, scratch=False):
                         traceback.print_exc()
                         self.logger.error(f'Unable to extract brew task information for {task_id}')
 
+    def update_konflux_db(self, build_info, success, scratch=False):
+        if scratch:
+            return
+
+        if not self.runtime.konflux_db:
+            self.logger.warning('Konflux DB connection is not initialized, not writing build record to the Konflux DB.')
+            return
+
+        self.logger.info('Storing build info in Konflux for Brew build %s', build_info['nvr'])
+        try:
+            source_repo, commitish = build_info['source'].split('#')
+
+            build_record = KonfluxBuildRecord(
+                name=self.metadata.name,
+                group=self.runtime.group,
+                version=build_info['version'],
+                release=build_info['release'],
+                assembly=self.runtime.assembly,
+                el_target=f'el{self.metadata.branch_el_target()}',
+                arches=self.metadata.get_arches(),
+                rpms=[],  # TODO
```

Comment:
We will need this before we can implement ocp4_scan with these records. 
A better name would be `installed_packages` . `BrewBuildInspector.get_all_installed_package_build_dicts` can return these for a given image.

Reply:
Done in f7d71194840b46fd2c567f4c6fb9987becd90392